server/character.lua

```diff
@@ -6,6 +6,10 @@ end
 
 ---@param source Source
 local function giveStarterItems(source)
+    while not exports.ox_inventory:GetInventory(source) do
+        Wait(100)
+    end
+    
```

Comment:
Use https://overextended.dev/ox_lib/Modules/WaitFor/Shared

Reply:
We want waitfor in there because it makes sure an infinite loop is not possible using the timeout, try to get it to work and if you really can't, we'll get an alternative

Reply:
You can mimick the code of waitFor to wait longer and to check using `not` instead of nil explicitly in here

Reply:
An alternative solution would be to register a net event for https://overextended.dev/ox_inventory/Events/Client#ox_inventoryupdateinventory on the client and use it to trigger a custom qbox server event which stores a global value that the inventory is loaded, and then you can wait for that global variable.